run.py

```diff
@@ -106,6 +123,30 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        if os.path.isfile(os.path.join(os.getcwd(), 'issues.db')):
+            issue_engine = create_engine('sqlite:///' + os.path.join(
+                app.config['BASE_DIR'], 'issues.db'))
+            issue_session_maker = sessionmaker(autocommit=False,
+                                               autoflush=False,
+                                               bind=issue_engine)
+            issue_db = scoped_session(issue_session_maker)
+            # Take a backup if issues.db has data dump.
+            if issue_db().execute('select count(*) from webcompat_issues').fetchall()[0][0] > 0:
+                if not os.path.exists(app.config['BACKUP_DEFAULT_DEST']):
+                    print 'Backup db folder does not exist'
+                    os.makedirs(app.config['BACKUP_DEFAULT_DEST'])
+                else:
+                    print 'Backup db folder exists'
+                time_stamp = time.strftime('%Y-%m-%dT%H-%M-%S', time.localtime())
+                issue_backup_db = 'issues_' + time_stamp + '.db'
+                os.rename(os.getcwd() + '/issues.db', app.config['BACKUP_DEFAULT_DEST'] + issue_backup_db)
```

Comment:
> Then we will have the updated schema. I will write for this code in a separate issue.

We decided to not do that if I remember correctly. See my mail
https://groups.google.com/forum/#!topic/mozilla.compatibility/f7tJmi9IB40